help/md/dnCBDSP.md

```diff
@@ -1,9 +1,38 @@
 ## name
 dnCBDSP
 ## title
+Conditional birth-death shift process
 ## description
+Simulates a tree under a birth-death process with shifts in birth and death rates.
 ## details
+This function simulates a tree under a birth-death process. The initial 
+birth and death rates can be specified with the rootAge and rootLambda arguments.
```

Comment:
Surely you mean `rootLambda` and `rootMu` rather than `rootAge` and `rootLambda`?

Also, I would prefer to call `dnCBDSP` a "distribution" rather than a "function" or a "simulator".